src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -177,6 +178,9 @@
 
     private final List<Allele> diploidNoCallAlleles = Arrays.asList(Allele.NO_CALL, Allele.NO_CALL);
 
+    // Number of called alleles
+    int calledAlleles = 0;
```

Comment:
actually, remove this field. It is state that is *not* carried over `apply` calls and thus it is useless. Turn it into a temp and pass directly to methods that need it